judge/utils/tests/test_cachedict.py

```diff
@@ -0,0 +1,20 @@
+from django.test import SimpleTestCase
+
+from judge.utils.cachedict import CacheDict
+
+
+class CacheDictTestCase(SimpleTestCase):
+    def setUp(self):
+        self.test_dict = {'a': True}
+        self.cache_dict = CacheDict(lambda id: self.test_dict[id])
+
+    def test_original_item_in_cachedict(self):
+        self.assertFalse('a' in self.cache_dict)
```

Comment:
does `self.assertNotIn` not work here (and the other places below)?

Reply:
Fixed